Tests/AsyncHTTPClientTests/HTTPClientTests.swift

```diff
@@ -215,6 +215,14 @@ class HTTPClientTests: XCTestCase {
         XCTAssertNil(url10)
     }
 
+    func testBadUnixWithBaseURL() {
+        let badUnixBaseURL = URL(string: "/foo", relativeTo: URL(string: "unix:")!)!
+        XCTAssertEqual(badUnixBaseURL.baseURL?.path, "")
+        XCTAssertThrowsError(try Request(url: badUnixBaseURL), "should throw") { error in
```

Comment:
Smallest nit ever seen: The message doesn't add any information.
```suggestion
        XCTAssertThrowsError(try Request(url: badUnixBaseURL)) { error in
```

Reply:
Sure, this is how the ones in `testBadRequestURI` are, but you're right that it doesn't add anything.